### PR TITLE
[Task] Fix search and filter

### PR DESCRIPTION
**Pull request by widyaput**
## Story/Task

_Fix search and filter_

## Summary

-   _Fix search and add filter_

## Test Plan

-   [ ] _Will start at the homepage_
-   [ ] _Fill query in navbar search and click search_
-   [ ] _Check filter tags_

## Screenshot

Screenshot from 2022-08-04 09-49-09
Screenshot from 2022-08-04 09-49-01

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6909,6 +6909,17 @@ fuse.js@^6.6.2:
   version "6.6.2"
   resolved "https://registry.yarnpkg.com/fuse.js/-/fuse.js-6.6.2.tgz#fe463fed4b98c0226ac3da2856a415576dc9a111"
   integrity sha512-cJaJkxCCxC8qIIcPBF9yGxY0W/tVZS3uEISDxhYIdtk8OL93pe+6Zj7LjCqVV4dzbqcriOZ+kQ/NE4RXZHsIGA==
+<<<<<<< HEAD
```
Comment (IloveNooodles) [inline yarn.lock:6912]: ini kok masi ada headbgaa?

File: test.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const Fuse = require('fuse.js');
```
Comment (IloveNooodles) [inline test.js:1]: ini bisa dihapus gaa ya test.js nya?

**Review by IloveNooodles — APPROVED** [review]
approved with dome comment

File: yarn.lock (hunk 1 above)
Comment (widyaput) [reply yarn.lock:6912]: awikwok ngantuk y

File: test.js (hunk 2 above)
Comment (widyaput) [reply test.js:1]: ah iya lupa anjir

**Pull request merged**